modules/bending_energy_diffusion_loss_notes.ipynb

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
<p>I just noticed, there should be a $\frac{1}{|\Omega|}$ factor in the definition of these loss functions right?</p><p> </p><p>Maybe you omitted it for simplicity, but it affects the scaling law which is what this notebook is about, so it might be good to include it here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>